crates/vm-superio-ser/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+# v0.3.0
+
+## Changed
+
+- Updated vmm-sys-util dependency to 0.11.0
```

Comment:
Can we keep the release commits separate? That's what we've done for the previous releases and I think it makes a bit more sense.